ee/modules/600-flant-integration/templates/pricing/config.yaml

```diff
@@ -11,6 +11,24 @@ prometheus:
     host_filter: false
     max_wal_time: 360h
     scrape_configs:
+    - job_name: 'deckhouse-telemetry'
+      params:
```

Comment:
I don't think this is required. You can safely delete the params section.

Reply:
removed